includes/class-wcpdf-settings.php

```diff
@@ -746,6 +747,17 @@ public function move_setting_after_id( $settings, $insert_settings, $after_setti
 		return $new_settings;
 	}
 
+	/**
+	 * Checks if guest access is enabled
+	 * 
+	 * @return bool
+	 */
+	public function is_guest_access_enabled() {
+		// return bool with filter applied
```

Comment:
Jhonny, this is not very important, but I think this comment is not necessary.

Reply:
Done

Reply:
You marked this as resolved, but the comment is still there.

Reply:
By the way, this change doesn't require a new review 😉

Reply:
Done